docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport Database Access
+---
+
+Access to AWS DynamoDB can be provided by [Teleport Database
+Access](../introduction.mdx). This allows for
+fine-grain access control through [Teleport's RBAC](../../database-access/rbac.mdx).
+
+This guide will help you to:
+
+- Install the Teleport Database Service.
+- Set up the Teleport Database Service to access DynamoDB.
+- Connect to your DynamoDB databases through the Teleport Database Service.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/aws-dynamodb_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/aws-dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A running Teleport cluster. If you don't already have Teleport running, see
+  [How to Choose a Teleport Edition](../../choose-an-edition/introduction.mdx)
+  to learn more about your options, and how to set up Teleport.
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- `aws` Command Line Interface (CLI) tool installed in PATH.
+- A host, e.g., an EC2 instance, where you will run the Teleport Database Service.
+  This guide assumes an EC2 instance when creating and applying IAM roles, and
+  must be adjusted accordingly for custom configurations.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow
+one of our [getting started guides](../getting-started.mdx) or try our Teleport
+Database Access [interactive learning track](https://play.instruqt.com/teleport/invite/imz8g1n1xzru).
+</Admonition>
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and
+`*.teleport.example.com`. You can substitute the address of your Teleport Proxy
+Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
```

Comment:
```suggestion
We will assume your Teleport cluster is accessible at `teleport.example.com`.
You can substitute the address of your Teleport Proxy Service. (For Teleport Cloud
customers, this will be similar to `mytenant.teleport.sh`.)
```